scrapinghub/client/items.py

```diff
@@ -59,3 +59,30 @@ def _modify_iter_params(self, params):
         if offset:
             params['start'] = '{}/{}'.format(self.key, offset)
         return params
+
+    def iter_by_chunks(self, chunksize=10000, *args, **kwargs):
```

Comment:
I have no idea what might be the best method name for this, so please suggest if you have something in mind.

Reply:
`job.items.list_iter(n=10_000)`
`job.items.iter_by(n=10_000)`
`job.items.iter_by_chunks(chunksize=10_000)`
I like the first two.

Reply:
I like the first one @manycoding as it's giving a better context of what's being returned overall, which is a generator of `list`s.

Reply:
I'll omit the **PEP 515** for now for backwards compatibility. It would be a great addition in 2020 nonetheless, when Python2 is deprecated.